tests/test_beerapi.py

```diff
@@ -1,28 +1,30 @@
+"""
+    This test is for the purpose of testing beerapi module with pytest.fixture and assuring requests works
+"""
 import pytest
 import requests
 import requests_mock
 import src.beerapi as beerapi
 
 
-@pytest.fixture(params= [requests.exceptions.ConnectTimeout, requests.exceptions.ConnectionError])
+@pytest.fixture(params=[requests.exceptions.ConnectTimeout, requests.exceptions.ConnectionError])
 def expected_exception_on_read(request):
     with requests_mock.Mocker() as session:
         session.get('http://www.tanczos.com/tanczos.com/beerinventory/webexport.csv', exc=request.param)
```

Comment:
This and the similar line around 25 get flagged by pylint for character count as well. I notice they both use the tanczos string, why don't we assign that to a variable and use the variable in these 2 lines. Once this and the above comment are fixed we should be pylint compliant.